app/src/main/java/org/dslul/openboard/inputmethod/latin/spellcheck/AndroidWordLevelSpellCheckerSession.java

```diff
@@ -110,12 +112,47 @@ public void onChange(boolean self) {
         cres.registerContentObserver(Words.CONTENT_URI, true, mObserver);
     }
 
+    private void updateLocale() {
```

Comment:
This can be moved in onCreate() if we remove updateLocale() calls from the two other methods

Reply:
Yeah that's right. I only created this method for these updateLocale() calls everywhere